src/test/java/com/exasol/bucketfs/testutil/LogBasedBucketFsMonitor.java

```diff
@@ -0,0 +1,66 @@
+package com.exasol.bucketfs.testutil;
+
+import static com.exasol.containers.ExasolContainerConstants.BUCKETFS_DAEMON_LOG_FILENAME_PATTERN;
+import static com.exasol.containers.ExasolContainerConstants.EXASOL_CORE_DAEMON_LOGS_PATH;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.bucketfs.ReadOnlyBucket;
+import com.exasol.bucketfs.monitor.BucketFsMonitor;
+import com.exasol.bucketfs.monitor.TimestampState;
+import com.exasol.clusterlogs.LogPatternDetector;
+import com.exasol.clusterlogs.LogPatternDetectorFactory;
+
+/**
+ * This class is basically a copy of the same class in project "exasol-testcontainers".
+ *
+ * <p>
+ * As soon as project "exasol-testcontainers" has been migrated to bucketfs-java 2.3.0 and has released a newer version
+ * v2, this class can be replaced by upgrading dependency exasol-testcontainers to a version v2 and using the class
+ * LogBasedBucketFsMonitor from exasol-testcontainers.
+ * </p>
```

Comment:
If not already done: create a ticket for this.

Reply:
Thanks - excellent proposal, see https://github.com/exasol/bucketfs-java/issues/46